scopes/react/react/react.env.ts

```diff
@@ -264,22 +264,25 @@ export class ReactEnv implements Environment {
     return {
       dependencies: {
         react: '-',
-        'core-js': '3.8.3',
+        'react-dom': '-',
       },
       // TODO: add this only if using ts
       devDependencies: {
-        '@types/node': '12.20.4',
-        '@types/react': '16.9.43',
-        '@types/jest': '26.0.20',
+        react: '-',
+        'react-dom': '-',
         '@types/mocha': '-',
-        '@types/react-router-dom': '5.1.7',
```

Comment:
`@types/react-router-dom` is irrelevant.
Bit will automatically add it when a component has dependency on `react-router-dom`.

It was probably added when automatic detection failed, and has been fixed since